### DISCUSSION TITLE
UserGuilds response missing features array

### DISCUSSION THREAD

**Discussion by alfonsohh99** [bwmarrin/discordgo#1221] [Q&A] [answered]
Hello everyone I would like to make use of the features array present in the discord response but as you can see here, this attribute is missing in the library's model so it is not returned to the user.
Discord's API response:
![dllhost_wdjZdddSMd](https://user-images.githubusercontent.com/47463368/184589025-280902ff-83f9-4a56-831a-9fe8bab70ff9.png)
*breaking point location: github.com\bwmarrin\discordgo@v0.25.0\restapi.go:366

discordgo model:
![Code_c291kklDYR](https://user-images.githubusercontent.com/47463368/184589367-287a3ce1-9049-4ac3-aca1-1f450f685222.png)
*model location: github.com\bwmarrin\discordgo@v0.25.0\structs.go:1027

How can I get the features array without modifying discordgo locally? Should the model be updated?

**Comment by topi314** [comment, marked as answer]
you can open a pr with this change

**Reply by alfonsohh99** [reply]
#1222
